packages/dart/noports_core/lib/src/common/streaming_logging_handler.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:async';
+
+import 'package:at_utils/at_logger.dart';
+import 'package:logging/logging.dart';
+
+class StreamingLoggingHandler implements LoggingHandler {
```

Comment:
A new at_logger LoggingHandler so we have a way to provide a stream of log messages